### ISSUE TITLE
flat/contents-typescript configs miss jsdoc/text-escaping { escapeHTML: true }

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [gajus/eslint-plugin-jsdoc#1307]
## Expected behavior

`configs['flat/contents-typescript'].rules['jsdoc/text-escaping']` should have value `['warn', { escapeHTML: true }]`.

## Actual behavior

It's just `'warn'` right now.

## ESLint Config

```js
import jsdoc from "eslint-plugin-jsdoc";

export default [jsdoc.configs["flat/contents-typescript"]];

```

## ESLint sample

```js
/**
 * Example.
 */
export const value = '';
```

## Environment

- Node version: 22.4.1
- ESLint version 9.8.1
- `eslint-plugin-jsdoc` version: 50.2.0
